dataviz/dataviz-in-practice.qmd

```diff
@@ -315,12 +315,12 @@ gapminder |>
   filter(year == past_year & !is.na(gdp)) |>
   ggplot(aes(dollars_per_day)) + 
   geom_histogram(binwidth = 1, color = "black") +
-  scale_x_continuous(trans = "log2")
+  scale_x_continuous(transform = "log2")
```

Comment:
I noticed in Help that this is deprecated; should probably be updated throughout.